include/megaapi_impl.h

```diff
@@ -743,6 +745,7 @@ class MegaTransferPrivate : public MegaTransfer, public Cachable
         long long endPos;
         int retry;
         int maxRetries;
+        bool foreignOverquota;
```

Comment:
Rather than declaring a new member (remember, prefix them with `m`), you should be able to recycle an existing one such as `streamingTransfer` and add the corresponding documentation to the related method/s